docs_dev/assemblies/development_environment.adoc

```diff
@@ -330,15 +318,25 @@ Create a Barbican secret
 openstack secret store --name testSecret --payload 'TestPayload'
 ```
 
-== Performing the Data Plane Adoption
+== Performing the adoption procedure
+
+To make our life easier when performing the adoption procedure, we can
```

Comment:
I suggest something like: "To simplify the adoption procedure, copy the deployment passwords that you use in the..."